oracle_gis.c

```diff
@@ -739,7 +745,30 @@ ewkbHeaderFill(oracleSession *session, ora_geometry *geom, char * dest)
 	return dest;
 }
 
+unsigned
+ewkbIsMeasured(oracleSession *session, ora_geometry *geom)
+{
+	unsigned gtype = 0;
+	if (geom->indicator->sdo_gtype == OCI_IND_NOTNULL)
+		numberToUint(session->envp->errhp, &(geom->geometry->sdo_gtype), &gtype);
+	switch ((gtype % 1000) / 100)
+	{
+		case 0:
+			return 0;
+		case 3:
+			return 1;
+		default:
+			oracleError(FDW_ERROR, "error converting SDO_GEOMETRY to geometry: measure dimension must be \"0\" or \"3\"");
+			return 0;  /* keep the compiler happy */
+	}
+}
 
+/*
```

Comment:
That makes a lot of sense; I forgot that "EWKB" stands for something.

I think I'll got with ``sdoDimension``.